art/estimators/classification/mxnet.py

```diff
@@ -368,6 +368,12 @@ def loss_gradient(self, x, y, **kwargs):
 
         return grads
 
+    def custom_gradient(self, nn_function):
+        return
+
+    def get_input_layer(self):
+        raise NotImplementedError
+
```

Comment:
Please remove these lines because `custom_gradient` is so far only part of `KerasClassifier` and not the Estimator API.
```suggestion
```